### PR TITLE
Updates for consistency

### PR DESCRIPTION
**Pull request by dralth**
Excellent start guys, thanks for putting the stuff together so far!

**The  things I changed**

1. Added [prettier](https://prettier.io/) it's a code formatter that has plugins for most text editors. I recommend trying it since with it you don't have to worry about spacing, capitalization, commas etc.
I run it on every save with the VSCode plugin. Brackets is not supported however, but you can run prettier from the command line once you finished or leave it to me.
This means I reformatted all the code so sorry about the big diff that you have to review. :P

2. I made sure headers and footers are the same on every page.

3. Fixed missing or duplicate closing tags.

4. Moved text into paragraph tags and removed any empty paragraph tags. We can add spacing later with css. The general rule of thumb is that html should be used for content only and css for all the styling.

5. Moved js to files and loading them at the end of `</body>` This makes sure dom is ready by the time they want to change it.

### CODE REVIEW

File: assets/css/siteWide/style.css [hunk 1]
```diff
@@ -1,17 +1,15 @@
 body {
-    background-color: white !important;
```
Comment (dralth) [inline assets/css/siteWide/style.css:2]: I am not sure we actually need `!important` here, but we can think about this next week.

File: assets/js/mixitUp/style.css [hunk 2]
```diff
@@ -1,200 +0,0 @@
-html,
```
Comment (dralth) [inline assets/js/mixitUp/style.css:1]: This was just a duplicate so I removed it.

File: assets/css/siteWide/style.css (hunk 1 above)
Comment (JohnGerhardt) [reply assets/css/siteWide/style.css:2]: Yep I agree, Was messing around with something and think I forgot to remove it...

File: assets/js/mixitUp/style.css (hunk 2 above)
Comment (JohnGerhardt) [reply assets/js/mixitUp/style.css:1]: ok!

File: attractions.html [hunk 3]
```diff
@@ -1,170 +1,260 @@
...
+    <title>Savannah Safari Attractions - Attractions and restaurants</title>
 
     <!--    css styles to be loaded -->
-    <link rel="stylesheet" href="assets/css/siteWide/style.css?v=1234234234">
+    <link rel="stylesheet" href="assets/css/siteWide/style.css?v=1234234234" />
 
     <!--    css styles for mix it up to be loaded -->
-    <link href="assets/css/mixitUp/reset.css" rel="stylesheet" />
-    <link href="assets/css/mixitUp/style.css" rel="stylesheet" />
+    <!-- <link href="assets/css/mixitUp/reset.css" rel="stylesheet" /> -->
```
Comment (dralth) [inline attractions.html:16]: I will comment these back shortly. It was just easier to compare pages and see inconsistencies. Either way it's not required for this week yet anyway.

Comment (JohnGerhardt) [reply attractions.html:16]: Ok

File: index.html [hunk 4]
```diff
@@ -1,51 +1,103 @@
...
+        </a>
+        <a href="https://www.twitter.com/savannahsafari">
+          <img class="socialMediaLogos" alt="twitter" src="assets/images/social/twitter.png" />
+        </a>
+        <a href="https://www.instagram.com/savannahsafari">
+          <img class="socialMediaLogos" alt="instagram" src="assets/images/social/instagram.png" />
+        </a>
+      </section>
+
+      <section id="footer-siteMap" class="footer siteMap">
```
Comment (dralth) [inline index.html:79]: Tbh, the classes are not necessary yet.

File: readme.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# Savannah Safari Attractions - UoL Team Web Dev 41
+
+## Setup
+
+To lunch a server locally you can run `python -m http.server` and visit http://0.0.0.0:8000/
+Alternatively you can use [VSCode + Live server extension](https://github.com/ritwickdey/vscode-live-server) or [Brackets](http://brackets.io/)
```
Comment (dralth) [inline readme.md:6]: I added a link to this plugin if you decide to try VSCode, but miss the live preview from Brackets.

File: index.html (hunk 4 above)
Comment (JohnGerhardt) [reply index.html:79]: Yeah I know but though it would be ok to get ahead a bit....

File: readme.md (hunk 5 above)
Comment (JohnGerhardt) [reply readme.md:6]: Will give it a go next week...

**Comment by dralth** [comment]
There's a setting now on github to hide the whitespace related differences that I caused
image

**Pull request merged**